crates/ibc/src/core/ics04_channel/handler/chan_open_init.rs

```diff
@@ -74,6 +74,8 @@ pub(crate) fn process<Ctx: ChannelReader>(
     let chan_end_on_a = ChannelEnd::new(
         State::Init,
         msg.ordering,
+        // Note: the counterparty channel Id passed by the relayer alternated to `None`,
+        // left for the `chan_open_try` handler to set
```

Comment:
Similarly not needed

Reply:
[e2ea605](https://github.com/cosmos/ibc-rs/pull/360/commits/e2ea6054fee2be83cd9421fce8b0b21bd92e754b)